libctru/include/3ds/services/y2r.h

```diff
@@ -1,113 +1,361 @@
+/**
+ * @file y2r.h
+ */
 #pragma once
 #include <3ds/types.h>
 
+/**
+ * @brief Input color formats
+ *
+ * For the 16-bit per component formats, bits 31-16 are padding and 15-0 contains the value.
```

Comment:
In what context is this relevant? The output is written as 16-bit values so there's no padding.